### ISSUE TITLE
Looking for maintainers

### ISSUE THREAD

**Issue by trezy** [fdgt-apis/api#675]
Im really happy to have been able to help so many people with their Twitch bots. To date, we've handled...

* 491,282 connections
* 46,495,879 incoming messages
* 90,248,509 outgoing messages
* USD$56,652,374.88 worth of subs and bits

That's **bonkers**. I never thought `fdgt` would see that kind of use, but I'm *really* glad that it has.

That said, I don't actually stream to Twitch anymore (I'm on YouTube now!), and my motivation to make any changes or updates to `fdgt` has been in the pits for quite a while. This doesn't mean the project is derelict or that it's going away, just that I won't be fixing bugs or adding new features.

However, this is open source! I'd love to find a new maintainer (or preferably several!) willing to take over the project. I'm happy to help get everybody up–and–running with project management, architectural and infrastructure diagrams, and general project guidelines. I'll also keep running and paying for all of the back end infrastructure as long as is necessary.

## What kind of changes are we looking at?

The biggest thing would be adding all of the events that are still missing from the Twitch API. There have been several added since the last time I worked on `fdgt`.

The other biggest request we've gotten is for the ability to have multiple connections to your sandbox.

## What does being a maintainer mean?

For `fdgt`, being a maintainer means...

* **Keeping the project up–to–date**
    There are several packages that need to be updated. None of them pose real security issues, but some of them could provide performance improvements.
* **Handling feature requests and bug reports**
    This is pretty much the case for any open source library. There are a handful of reports in the issues on this repo that I just never got around to.
* **Taking care of the community**
    There's a small community around `fdgt`, but they've mostly gone inactive since I've stopped actively managing it. I expect any maintainer to be willing to handle this community with care.

## TL;DR

I'm not streaming to Twitch anymore so I'm not actively maintaining `fdgt`. I'd like to find somebody (or multiple somebodys) to take over maintenance of the project.

**Comment by GregHattJr** [comment]
I don't know that I have the bandwidth to care for the whole project myself but I'm definitely willing to give a hand where I can. I can definitely handle things like code review, issue grooming, updating dependencies and that sort of thing. If you'd like to talk more about it I'd be happy to talk.
